Cargo.toml

```diff
@@ -48,6 +50,7 @@ net = { path = "./net", package = "dataplane-net", features = ["test_buffer"] }
 pipeline = { path = "./pipeline", package = "dataplane-pipeline" }
 rekon = { path = "./rekon", package = "dataplane-rekon" }
 routing = { path = "./routing", package = "dataplane-routing" }
+test-utils = { path = "./test-utils", package = "dataplane-test-utils" }
 vpc-manager = { path = "./vpc-manager", package = "dataplane-vpc-manager" }
```

Comment:
Didn't we get rid of the `vpc-manager` crate?

Reply:
yup, this is an error.  Will fix.